test/method/testbader.py

```diff
@@ -56,3 +57,31 @@ def test_val(self):
         refData = [6.9378, 0.3639, 0.3827]  # values from `bader` package
 
         assert_allclose(self.analysis.fragcharges, refData, atol=0.15)
+
+    def test_chgsum_hf(self):
+        """Does the sum of charges equate to the number of electrons for a simple molecule?"""
+
+        hfpath = os.path.dirname(os.path.realpath(__file__)) + "/hf.out"
+        self.data = ccread(hfpath)
```

Comment:
I don't think any of the new variables created in both of these methods need to be part of `self`, since they're only used in their respective methods.

Reply:
Yes, I agree. I've changed the variables not to be part of self.